src/main/resources/data/modern_industrialization/recipes/vanilla_recipes/assembler/chest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "type": "modern_industrialization:assembler",
+  "eu": 8,
+  "duration": 200,
+  "item_inputs": {
+      "tag": "minecraft:planks",
```

Comment:
Will conflict with a lot of thing

Reply:
Yeah, I'll try to cook up something for it.

Reply:
Went with adding iron parts into the chest.
2 bolts to represent hinges and an iron ring for the clasp.